modules/solid_mechanics/doc/content/source/vectorpostprocessors/AverageSectionValueSampler.md

```diff
@@ -27,14 +27,29 @@ This postprocessor should only be used for regularly meshed geometries as it rel
 same weights on the cross section. There is no check within the code for this condition, so the user must
 ensure that the mesh is reasonably uniform.
 
+Because this VectorPostprocessor is intended to be applied to geometries with extruded meshes, with layers
+of nodes that all occupy the same axial position, it will generate an error if the numbers of nodes located
+at the various axial positions vary. However, there are valid scenarios when this is not the case. For example,
+if part of the extruded mesh is refined, the number of nodes per layer will differ along the axis of the
+extrueded section. The [!param](/VectorPostprocessors/AverageSectionValueSampler/require_equal_node_counts) parameter can be set to `false` in such situations to skip
+that check. The number of nodes per position is reported in the ouputs as the `node_count` vector to allow the
+user to ensure that each layer contains all expected nodes.
+
+The optional [!param](/VectorPostprocessors/AverageSectionValueSampler/symmetry_plane) parameter is intended to be used when there is a symmetry plane passing through
+the extruded section being evaluated. If there is volumetric expansion in the section and this is used to 
```

Comment:
```suggestion
the extruded section being evaluated, and through the origin of the frame of reference. If there is volumetric expansion in the section and this is used to 
```

Reply:
I'm not sure what you mean about the origin. It doesn't matter where the origin is for this.